routing/routing_integration_tests/routing_test_tools.cpp

```diff
@@ -19,6 +17,9 @@
 
 #include "indexer/index.hpp"
 
+#include "geometry/distance_on_sphere.hpp"
+#include "geometry/latlon.hpp"
```

Comment:
Please, group geometry inclusions after the group of inclusions from platform?

Reply:
Done